default.nix

```diff
@@ -5,7 +5,9 @@ in
 , config ? {}
 , dconfig ? "testnet_staging_full"
 , gitrev ? "unknown"
-, pkgs ? (import (localLib.fetchNixPkgs) { inherit system config; }) }:
+, pkgs ? (import (localLib.fetchNixPkgs) { inherit system config; })
+, enableProfiling ? false
```

Comment:
I'd add a comment here: `# profiling slows down performance by 50% so we don't enable it by default`